cclib/parser/molcasparser.py

```diff
@@ -34,9 +34,13 @@ def __repr__(self):
         """Return a representation of the object."""
         return 'Molcas("%s")' % (self.filename)
 
-    #These are yet to be implemented.
     def normalisesym(self, label):
-        """Does Molcas require symmetry label normalization?"""
+        """Normalise the symmetries used by Molcas.
+
+        The labels are standardized except for the first character being
+        lowercase.
```

Comment:
I don't think we are being strict about the 80 column rule, especially for comments and docstrings.